met-api/tests/unit/api/test_survey.py

```diff
@@ -39,9 +39,11 @@
 
 
 @pytest.mark.parametrize('survey_info', [TestSurveyInfo.survey1])
-def test_create_survey(client, jwt, session, survey_info):  # pylint:disable=unused-argument
+def test_create_survey(client, jwt, session, survey_info,
+                       setup_admin_user_and_claims):  # pylint:disable=unused-argument
```

Comment:
Do you know if we need to keep the unused argument around instead of adding a rule to ignore it?